lib/tracker_api/resources/story.rb

```diff
@@ -21,7 +21,7 @@ class Story
       attribute :integration_id, Integer
       attribute :kind, String
       attribute :label_ids, [Integer]
-      attribute :labels, [Label]
+      attribute :labels, [Label], default: nil
```

Comment:
The reason defaulting the labels attribute fixes this issue has to do with the `UpdateRepersenter` class that specifies that an empty collection should be rendered. This is done so that if you want to clear all the labels you can set `story.labels = []` and it will get rendered to json instead of ignored. 
`collection :labels, class: Label, decorator: Label::UpdateRepresenter, render_empty: true`

Here is some example code and output that shows the behavior.
```
st = Story.new(self.dirty_attributes)
st.labels
>nil
UpdateRepresenter.new(Story.new(st.dirty_attributes)).to_json
>{}
st.labels = []
[]
UpdateRepresenter.new(Story.new(st.dirty_attributes)).to_json
{"labels":[]}
```